### PR TITLE
metric client fix

### PR DESCRIPTION
**Pull request by edengal**

### CODE REVIEW

**Review by stasp44 — APPROVED** [review]

File: timbermill-java/timbermill-api/src/main/java/com/datorama/oss/timbermill/unit/Event.java [hunk 1]
```diff
@@ -288,7 +288,7 @@ private int getNumberMapSize(Map<String, Number> map) {
 					size += entry.getKey().length();
 				}
 				if (entry.getValue() != null) {
-					size += (Math.log10(entry.getValue().doubleValue()) + 1) + 4; // "":,
+					size += (entry.getValue().toString().length() + 1) + 4; // "":,
```
Comment (janevelger) [inline timbermill-java/timbermill-api/src/main/java/com/datorama/oss/timbermill/unit/Event.java:291]: @edengal How big this value can be? entry.getValue().toString()
Please surround this code with exception handling.

**Review by janevelger — CHANGES_REQUESTED** [review]
Added comment

File: timbermill-java/timbermill-api/src/main/java/com/datorama/oss/timbermill/unit/Event.java (hunk 1 above)
Comment (edengal) [reply timbermill-java/timbermill-api/src/main/java/com/datorama/oss/timbermill/unit/Event.java:291]: @janevelger How long the string will be? 1.7976931348623157E308 (Max double value)

Comment (edengal) [reply timbermill-java/timbermill-api/src/main/java/com/datorama/oss/timbermill/unit/Event.java:291]: @janevelger Can you elaborate on which exception handling you are referring to?

**Review by janevelger — APPROVED** [review]

**Pull request merged**